HttpCache.php

```diff
@@ -20,6 +20,8 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 
+@trigger_error('The '.__CLASS__.' class is deprecated since version 1.2 and will be removed in 2.0. Use FOS\HttpCacheBundle\SymfonyCache\EventDispatchingHttpCache instead.', E_USER_DEPRECATED);
```

Comment:
__CLASS__ is not defined outside of a class. Use something like this: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/Extension/Core/ChoiceList/ObjectChoiceList.php#L14